services/director-v2/docker/entrypoint.sh

```diff
@@ -13,6 +13,13 @@ INFO="INFO: [$(basename "$0")] "
 WARNING="WARNING: [$(basename "$0")] "
 ERROR="ERROR: [$(basename "$0")] "
 
+# Read self-signed SSH certificates (if applicable)
+#
+# In case the service must access a docker registry in a secure way using
+# non-standard certificates (e.g. such as self-signed certificates), this call is needed.
+# It needs to be executed as root.
+update-ca-certificates
```

Comment:
same question here,  the docker registry is accessed via the director right?

Reply:
adjusted the comment, for the director this was added anticipatory, I at this point dont know if or which outside services the direcot-v2 might need.

But for sure, the director-v2 will eventually have to pass the self-signed cert onwards to the dynamic-sidecars and dy-proxys, so I guess he needs it and it is ok if the director-v2 trusts this self-signed certificate

Reply:
I guess it is not acessing https services, but why would the director-v2 not trust a certificate it will instruct the dy-sidecars spawned by it to trust? :D maybe I am missing something here

Reply:
My indirect question was something on the lines of: what issues does this fix?